### ISSUE TITLE
Can you share the fine-tuned models weights?

### ISSUE THREAD

**Issue by hasan7n** [ai-center-kth/sentence-cubert#1]
Hi @mhagglun 

Thanks for this great work! Can you please share the final models weights after finetuning? Specifically, the model trained with triplet-loss and K-means (COCLUBERT-Triplet with K-means).

Thanks!
